src/hb-subset.h

```diff
@@ -40,51 +40,56 @@ HB_BEGIN_DECLS
  * the produced subset. If glyphs are dropped their indices will be retained
  * as an empty glyph. Defaults to false.
- * @HB_SUBSET_FLAG_DESUBROUTINIZE: If set and subsetting a CFF font the
+ * @HB_SUBSET_FLAGS_DESUBROUTINIZE: If set and subsetting a CFF font the
  * subsetter will attempt to remove subroutines from the CFF glyphs.
  * Defaults to false.
- * @HB_SUBSET_FLAG_NAME_LEGACY: If set non-unicode name records will be
+ * @HB_SUBSET_FLAGS_NAME_LEGACY: If set non-unicode name records will be
  * retained in the subset. Defaults to false.
- * @HB_SUBSET_FLAG_SET_OVERLAPS_FLAG:  If set the subsetter will set the
+ * @HB_SUBSET_FLAGS_SET_OVERLAPS_FLAG:  If set the subsetter will set the
  * OVERLAP_SIMPLE flag on each simple glyph. Defaults to false.
- * @HB_SUBSET_FLAG_PASSTHROUGH_UNRECOGNIZED: If set the subsetter will not
+ * @HB_SUBSET_FLAGS_PASSTHROUGH_UNRECOGNIZED: If set the subsetter will not
  * drop unrecognized tables and instead pass them through untouched.
  * Defaults to false.
- * @HB_SUBSET_FLAG_NOTDEF_OUTLINE: If set the notdef glyph outline will be
+ * @HB_SUBSET_FLAGS_NOTDEF_OUTLINE: If set the notdef glyph outline will be
  * retained in the final subset. Defaults to false.
- * @HB_SUBSET_FLAG_GLYPH_NAMES: If set the PS glyph names will be retained
+ * @HB_SUBSET_FLAGS_GLYPH_NAMES: If set the PS glyph names will be retained
  * in the final subset. Defaults to false.
- * @HB_SUBSET_FLAG_NO_PRUNE_UNICODE_RANGES: If set then the unicode ranges in
+ * @HB_SUBSET_FLAGS_NO_PRUNE_UNICODE_RANGES: If set then the unicode ranges in
  * OS/2 will not be recalculated.
- * @HB_SUBSET_FLAG_RETAIN_ALL_FEATURES: If set all layout features will be
+ * @HB_SUBSET_FLAGS_RETAIN_ALL_FEATURES: If set all layout features will be
  * retained. If unset then the set accessed by
  * hb_subset_input_layout_features_set() will be used to determine the features
  * to be retained.
+ * HB_SUBSET_FLAGS_ALL: bit set with all flags set.
  *
  * List of boolean properties that can be configured on the subset input.
  *
  * Since: REPLACE
  **/
 typedef enum
 {
-  HB_SUBSET_FLAG_HINTING = 0,
-  HB_SUBSET_FLAG_RETAIN_GIDS,
-  HB_SUBSET_FLAG_DESUBROUTINIZE,
-  HB_SUBSET_FLAG_NAME_LEGACY,
-  HB_SUBSET_FLAG_SET_OVERLAPS_FLAG,
-  HB_SUBSET_FLAG_PASSTHROUGH_UNRECOGNIZED,
-  HB_SUBSET_FLAG_NOTDEF_OUTLINE,
-  HB_SUBSET_FLAG_GLYPH_NAMES,
-  HB_SUBSET_FLAG_NO_PRUNE_UNICODE_RANGES,
-  HB_SUBSET_FLAG_RETAIN_ALL_FEATURES,
-} hb_subset_flag_t;
+  HB_SUBSET_FLAGS_NONE = 0,
+  HB_SUBSET_FLAGS_NO_HINTING = 1,
+  HB_SUBSET_FLAGS_RETAIN_GIDS = 1 << 1,
+  HB_SUBSET_FLAGS_DESUBROUTINIZE = 1 << 2,
+  HB_SUBSET_FLAGS_NAME_LEGACY = 1 << 3,
+  HB_SUBSET_FLAGS_SET_OVERLAPS_FLAG = 1 << 4,
+  HB_SUBSET_FLAGS_PASSTHROUGH_UNRECOGNIZED = 1 << 5,
+  HB_SUBSET_FLAGS_NOTDEF_OUTLINE = 1 << 6,
+  HB_SUBSET_FLAGS_GLYPH_NAMES = 1 << 7,
+  HB_SUBSET_FLAGS_NO_PRUNE_UNICODE_RANGES = 1 << 8,
+  HB_SUBSET_FLAGS_RETAIN_ALL_FEATURES = 1 << 9,
```

Comment:
Please tabulate the `=` signs using enough tabs.

In other places we use hex eg `0x00000080u`. Maybe match that?